Makefile

```diff
@@ -1,18 +1,14 @@
 # This Makefile requires the following commands to be available:
 # * python3.6
-# * docker
-# * docker-compose
 
-DEPS:=requirements.txt
-DOCKER_COMPOSE=$(shell which docker-compose)
+DEPS=requirements.txt
 
-PIP:="venv/bin/pip"
-CMD_FROM_VENV:=". venv/bin/activate; which"
-TOX=$(shell "$(CMD_FROM_VENV)" "tox")
-PYTHON=$(shell "$(CMD_FROM_VENV)" "python")
+PIP="venv/bin/pip"
+TOX="venv/bin/tox"
+PYTHON="venv/bin/python"
```

Comment:
Slightly out of context, but I also like to have:
```mk
VIRTUAL_ENV ?= venv
```
And then use it with:
```mk
PYTHON=$(VIRTUAL_ENV)/bin/python
```
The idea is it leverage existing `VIRTUAL_ENV` variable, making it possible not only to override the folder easily, but also running from an already existing venv named differently

Reply:
good idea, but I need to test it thoroughly first and it is not that easy, so I will keep this improvement in mind but skip it for this time.

Reply:
Yes that should definitely be addressed in a different PR :+1:

Reply:
was thinking about this suggestion.
if we use `VIRTUAL_ENV` variable which has been set by `source ./venv/bin/activate`, we can get weird behavior if we activate virtual environment of one project and then run make target of another.

it would be difficult to reproduce and to debug as well, so I am not sure now if we actually need to support that or how to implement it properly.

Reply:
Definitely if people run it from another complex venv full of (eventually clashing) dependencies, that would make a mess. It's clearly a feature for people who know what there're doing